src/views/bav/confirm-details.html

```diff
@@ -0,0 +1,148 @@
+{% extends "base-form.njk" %}
+{# the content for this page is controlled by locales/en/default.yml #}
+{% set hmpoTitle = confirmDetails.title %}
+{% set hmpoPageKey = "confirmDetails" %}
+{% set gtmJourney = "bav - confirmDetails" %}
+
+{% from "govuk/components/header/macro.njk" import govukHeader %}
+{% from "govuk/components/summary-list/macro.njk" import govukSummaryList %}
+{% from "govuk/components/inset-text/macro.njk" import govukInsetText %}
+{% from "hmpo-form/macro.njk" import hmpoForm %}
+
+{% block mainContent %}
+
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-three-quarters">
+    <h1 id="confirm-details-header" class="gov" data-page="{{hmpoPageKey}}" class="govuk-heading-l">
+        {{ translate("confirmDetails.title") }}
+    </h1>
+  </div>
+</div>
+
+{{ govukSummaryList({
+  classes: 'govuk-!-margin-bottom-6',
+  rows: [
+    {
+      key: {
+        text: ""
+      },
+      value: {
+        text: ""
+      }
+    },
+    {
+      key: {
+        text: translate("confirmDetails.fullName")
+      },
+      value: {
+        text: fullName
+      }
+    },
+    {
+      key: {
+        text: translate("confirmDetails.sortCode")
+      },
+      value: {
+        html: sortCode
+      },
+      actions: {
+        items: [
+          {
+            href: "/enter-account-details/edit",
+            text: translate("confirmDetails.changeLink"),
+            visuallyHiddenText: translate("confirmDetails.sortCode")
+          }
+        ]
+      }
+    },
+    {
+      key: {
+        text: translate("confirmDetails.accountNumber")
+      },
+      value: {
+        html: accountNumber
+      },
+      actions: {
+        items: [
+          {
+            href: "/enter-account-details/edit",
+            text: translate("confirmDetails.changeLink"),
+            visuallyHiddenText: translate("confirmDetails.accountNumber")
+          }
+        ]
+      }
+    }
+  ]
+})
+}}
+
+{% set insetBody %}
+  {{- "confirmDetails.insetText" | translate -}}
+{% endset %}  
+
+{{ govukInsetText({
+  html: insetBody
+}) }}
+
+{% call hmpoForm(ctx) %}
+    {{ hmpoSubmit(ctx, {id: "submitDetails", text: translate("confirmDetails.submitDetails")}) }}
+{% endcall %}
+
+<p class="bav__instruction"><a href={{ "landingPage.confirmIdentityAnotherWay.abortLink" | translate }}> {{ "confirmDetails.noSubmitDetails" | translate }}</a></p>
+
+{% endblock %}
+
+{# generate the specific footer items required for the PYI flows #}
+{% set footerNavItems = translate("govuk.footerNavItems") %}
+
+{% block footer %}
+    {{ govukFooter( footerNavItems ) }}
+{% endblock %}
+
+{% block bodyEnd %}
+  {% block scripts %}
+    <script nonce="{{ cspNonce }}">
+
+<!--Disable enter button from submitting the form but allow if form submit button is focused-->
+    var form = document.getElementsByTagName('form')[0];
```

Comment:
Is it possible to put an ID on the from and use that? I know it's unlikely that we'll add another form to this page but it's better practise to be as specific as possible with selectors

Reply:
Oh yeah looking into this further because the button is in a form the enter key is treated like a click event, and this doesn't just have to be the enter key it will be whatever button is most appropriate for the user agent which sounds good 👍  https://www.w3.org/TR/2011/WD-html5-20110525/association-of-controls-and-forms.html#implicit-submission